Remotion/Web/Development.WebTesting.IntegrationTests/CommandControlObjectTest.cs

```diff
@@ -58,7 +58,7 @@ public void TestIsDisabled_SetMethodsThrow ()
       var control = home.Commands().GetByLocalID ("TestCommand3");
 
       Assert.That (control.IsDisabled(), Is.True);
-      Assert.That (() => control.Click(), Throws.Exception.Message.EqualTo (AssertionExceptionUtility.CreateControlDisabledException ("Command").Message));
+      Assert.That (() => control.Click(), Throws.Exception.Message.EqualTo (AssertionExceptionUtility.CreateControlDisabledException ("Click").Message));

```

Comment:
What was the reasoning being it being OK to replace "command" with "control"?

So, basically we'd have CreateControlDisabledException() and CreateCommandDisabledException(), they can then delegate to shared private code.

Reply:
@MichaelKetting  I could not see a reason why `The 'Command' is currently in a disabled state. Therefore, the 'Click' operation is not possible.` has informational benefits over `The control is currently in a disabled state. Therefore, the 'Click' operation is not possible.`.

If you want to keep the message as it was, I'll add the `CreateCommandDisabledException()` method.

Reply:
'Control' refers to a control object, 'Command' to a command owned by a control object. A command can be disabled even if the control (object) is enabled. therefor, this would be misleading.

Reply:
Makes sense, thanks